CONP_main_data_documentation.htm

```diff
@@ -21,13 +21,11 @@ <h2>1. Metadata</h2>
 
 
 
-The schema also includes many other recommended and optional fields.  The DATS dataset schema can be found <a href="https://github.com/biocaddie/DATS/blob/master/dataset_schema.json">here</a>, and details of CONP's expansions and descriptions of what to enter in each field <a href="../CONP_DATS_fields.docx">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
+The schema also includes many other recommended and optional fields.  The DATS dataset schema can be found <a href="https://github.com/biocaddie/DATS/blob/master/dataset_schema.json">here</a>, and details of CONP's expansions and descriptions of what to enter in each field <a href="https://github.com/emmetaobrien/conp-documentation/blob/master/CONP_DATS_fields.md">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
```

Comment:
```suggestion
The schema also includes many other recommended and optional fields. The DATS dataset schema can be found <a href="https://github.com/datatagsuite/schema/blob/master/dataset_schema.json">here</a>, and details of CONP's expansions and descriptions of what to enter in each field <a href="https://github.com/emmetaobrien/conp-documentation/blob/master/CONP_DATS_fields.md">here</a>. The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template. An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
```

Reply:
Plus, link to DATS schema should use datatagsuite no biocaddie (older version) 
Link to CONP-PCNO github should be relative not absolute. Currently point to emmetaobrien fork.